stdlib/public/core/FlatMap.swift

```diff
@@ -24,6 +24,26 @@ extension LazySequenceProtocol {
     FlattenSequence<LazyMapSequence<Elements, SegmentOfResult>>> {
     return self.map(transform).flatten()
   }
+  
+  /// Returns a `LazyMapSequence` containing the concatenated non-nil
+  /// results of mapping transform over this `Sequence`.  The elements of
+  /// the result are computed lazily, each time they are read.
```

Comment:
The second sentence could just be removed, since that's covered by the `Lazy...` type of the return.